manual/src/threelang/tfgrid/gatewayed_vms.md

```diff
@@ -0,0 +1,39 @@
+# Machines with Gateways Namespace
+
+- To deploy a number of virtual machines on the same network with a domain name refer to a running server on the machines, use the `gatewayed_vms` namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.gatewayed_vms.create
```

Comment:
I would get rid of this action and incorporate it into the one to deploy a machine. You can add two extra optional parameters backend_port and fqdn and then also a argument gateway_name. So as a user if you add gateway_name to the arguments it means you should add a gateway name to your machine. Then if you add the parameter fqdn it means you are adding a fully qualified domain name to the machine. If you provide none of those arguments it means you are just deploying a simple vm.

Reply:
will update the v client first, then update the docs